packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -193,6 +194,10 @@ class Plugin implements CLNPlugin {
           } else {
             param = HashMap();
           }
+          if (!param.containsKey('id')) {
+            var notification = notifications[jsonRequest.method]!;
+            await notification.call(this, param);
+          }
```

Comment:
Here do need to skip the rcp call

```dart
  if (!param.containsKey('id')) {
            var notification = notifications[jsonRequest.method]!;
            await notification.call(this, param);
    } else {
      /// make the RPC call
   }
```

In addition I preferer that you have a function like the `_call`

Reply:
So Like _callNotification() ?

Reply:
`handlingNotification` is a better name?

Reply:
Yes it is